packages/helpers/src/product/getProductTierPrices.ts

```diff
@@ -0,0 +1,32 @@
+import { Product } from "@shopware-pwa/commons/interfaces/models/content/product/Product";
+
+/**
+ * @beta
+ */
+interface TierPrice {
+  label: string; // information about the quantity range
+  quantity: number; // price limit value
+  unitPrice: number; // price for only one product in given price range (quantity)
+}
+
+/**
+ * Get the prices depending on quantity added to cart.
+ * Tier prices can be set in `Advanced pricing` tab in `Product view` (admin panel)
+ *
+ * @returns TierPrice[]
+ * @beta
+ */
+export function getProductTierPrices(
+  product: Product
+): TierPrice[] | undefined {
```

Comment:
can we always return an empty array as a fallback?
```suggestion
): TierPrice[] {
```

Reply:
:+1: you're right. fixed

Reply:
done